components/logstash/scripts/preconfigure_rabbitmq_properties.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+from cloudify import ctx
+
+
+def main():
+    """Get rabbitmq configuration needed to generate logstash.conf file."""
+    source = ctx.source.instance.runtime_properties
+    target = ctx.target.node.properties
+    props = [
+        'rabbitmq_endpoint_ip',
+        'rabbitmq_username',
+        'rabbitmq_password',
+    ]
+    for prop in props:
+        source[prop] = target[prop]
+
+
+if __name__ == '__main__':
```

Comment:
Why is this needed?
I think that by default these scripts are evaluated by script plugin..